File: week2/readability.c
#include <cs50.h>
#include <stdio.h>
#include<math.h>
#include<ctype.h>
#include <string.h>

int main(void)
{

    string text = get_string("TEXT:");

    int latters = 0;
    int words = 1;
    int sentences = 0;

    for (int i = 0; i < strlen(text);i++)
    {
        if(isalpha(text[i]))
        {
           latters++;
        }
        else if(text[i] == ' ')
        {
           words++;
        }
        else if( text[i] == '.'|| text[i] == '?'||text[i] =='!' )
        {
            sentences++;
        }
    }
      float L = (float) latters /(float) words * 100;
      float S = (float) sentences/(float) words * 100;

      int index = round(0.0588 * L - 0.296 * S - 15.8);

      if(index < 1)
      {
           printf("Before Grand 1\n");
      }
      else if(index > 16)
      {
           printf("Grade 16+\n");
      }
      else
      {
           printf("Grade %i\n",index);
      }

}

File: week6/readability.py
from cs50 import get_string

text = get_string("Text: ")

letter = 0
sentence = 0
word = 1

for i in range(len(text)):
    if text[i].isalpha():
        letter += 1
    elif text[i].isspace():
        word += 1
    elif text[i] in ['!', '.', '?']:
        sentence += 1

L = float(letter / word) * 100
S = float(sentence / word) * 100
index = round(0.0588 * L - 0.296 * S - 15.8)

if index <= 1:
    print("Before Grade 1")
elif index >= 16:
    print("Grade 16+")
else:
    print(f"Grade {index}")